easybuild/easyconfigs/a/AMPL-MP/AMPL-MP-3.1.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'AMPL-MP'
+version = '3.1.0'
+
+homepage = 'https://github.com/ampl/mp'
+description = """ An open-source library for mathematical programming. """
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['https://github.com/ampl/mp/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['587c1a88f4c8f57bef95b58a8586956145417c8039f59b1758365ccc5a309ae9']
+
+builddependencies = [
+    ('binutils', '2.28'),
+    ('CMake', '3.9.5'),
+]
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['lib/libasl.a', 'lib/libmp.a'],
```

Comment:
Why not also check for `lib/libbenchmark.a`?

I would include checking for specific binaries here too:

```python
sanity_check_paths = {
    'files': ['bin/tableproxy64', 'lib/libasl.a', 'lib/libbenchmark.a', 'lib/libmp.a'],
    'dirs': ['include/asl', 'include/benchmark', 'include/mp', 'share'],
}
```